db/migrate/20200511224704_add_query_friendly_admin_fields_to_articles.rb

```diff
@@ -0,0 +1,6 @@
+class AddQueryFriendlyAdminFieldsToArticles < ActiveRecord::Migration[5.2]
+  def change
+    add_column :articles, :query_friendly_title_preamble, :string
```

Comment:
I'm trying to think of a name here that would make it clear it's for search engines. 
- discovery_friendly_title_preamble
- optimized_search_title_preamble
- search_engine_title_preamble

Reply:
Good call. I like `optimized_search_title_preamble`

Reply:
Wonderful suggestions @mstruve: makes me realize how much of my hesitate was related to the naming of these fields!